homeassistant/components/conversation.py

```diff
@@ -29,20 +31,43 @@
     vol.Required(ATTR_TEXT): vol.All(cv.string, vol.Lower),
 })
 
-CONFIG_SCHEMA = vol.Schema({
-    DOMAIN: vol.Schema({}),
-}, extra=vol.ALLOW_EXTRA)
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({
+    cv.string: vol.Schema({
+        vol.Required('keywords'): cv.string,
+        vol.Required('action'): cv.SCRIPT_SCHEMA,
+    })
+})}, extra=vol.ALLOW_EXTRA)
 
 
 def setup(hass, config):
     """Register the process service."""
-    warnings.filterwarnings('ignore', module='fuzzywuzzy')
     from fuzzywuzzy import process as fuzzyExtract
+    warnings.filterwarnings('ignore', module='fuzzywuzzy')
```

Comment:
This one was above the import because fuzzywuzzy would generate warnings on import of the module.